lxd/db/projects.go

```diff
@@ -128,6 +128,28 @@ func (c *ClusterTx) ProjectHasImages(name string) (bool, error) {
 	return enabled, nil
 }
 
+func (c* ClusterTx) ProjectGetContainerLimit(name string) (string, error) {
+	project, err := c.ProjectGet(name)
+	if err != nil {
+		return "", errors.Wrap(err, "fetch project")
+	}
+
+	containerlimit := project.Config["limits.containers"]
+
+	return containerlimit, nil
+}
+
+func (c* ClusterTx) ProjectGetVMLimit(name string) (string, error) {
```

Comment:
I don't think we'll want a function per config key.
I'd instead suggest adding a new `load.go` file in `lxd/project` and defining a `GetByName` function in there that performs a simple `ProjectGet` and returns the `api.Project` if found.

When we need to check a config key, we can then just call that and look for `project.Config["blah"]` similarly to what we do in a number of places already.